config/database.yml

```diff
@@ -57,6 +58,7 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
+  url: <%= ENV['DATABASE_URL'] %>
```

Comment:
Could we do a `ENV.fetch` here with a default. Same with development.

Reply:
I guess, it would break the current local setup, which relies on locally running DB.

Reply:
Maybe I missed something? Is the url inside the container localhost too?

Reply:
Nope, it points to the `postgres` container: https://github.com/hopsoft/stimulus_reflex_expo/pull/23/files#diff-4e5e90c6228fd48698d074241c2ba760R35

Reply:
right, so in order to not break existing functionality, doesn't this need to be something like `<%= ENV.fetch("DATABASE_URL", "127.0.0.1:5432") %>` so that it will fallback to the correct address if that env variable is not set or am I not seeing something?

Reply:
Rails uses local socket connection if no URL is provided, which is different from specifying the `127.0.0.1`, which requires user/password.
Using `nil' is exactly the same as not specifying `url` at all (which is the current behaviour).